<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddressManagement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="addressmanagement.qrc">
    <normaloff>:/img/addressmanagement.png</normaloff>:/img/addressmanagement.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_Search">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_Search">
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset resource="addressmanagement.qrc">
          <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>229</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>229</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddressAdd"/>
   <addaction name="actionAddressDelete"/>
   <addaction name="actionGroupAdd"/>
   <addaction name="actionGroupDelete"/>
   <addaction name="actionUpload"/>
   <addaction name="actionDownload"/>
   <addaction name="actionDuplicateCheck"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddressAdd">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
   </property>
   <property name="text">
    <string>AddressAdd</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAddressDelete">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
   </property>
   <property name="text">
    <string>AddressDelete</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionGroupAdd">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/group_add.png</normaloff>:/img/group_add.png</iconset>
   </property>
   <property name="text">
    <string>GroupAdd</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionGroupDelete">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/group_delete.png</normaloff>:/img/group_delete.png</iconset>
   </property>
   <property name="text">
    <string>GroupDelete</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/upload.png</normaloff>:/img/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/download.png</normaloff>:/img/download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionDuplicateCheck">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/duplication.png</normaloff>:/img/duplication.png</iconset>
   </property>
   <property name="text">
    <string>DuplicateCheck</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPrinter">
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/printer.png</normaloff>:/img/printer.png</iconset>
   </property>
   <property name="text">
    <string>Printer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="addressmanagement.qrc"/>
 </resources>
 <connections/>
</ui>
